src/admin-views/settings/tickets-commerce/paypal/connect/inactive.php

```diff
@@ -13,12 +13,16 @@
  * @var TEC\Tickets\Commerce\Gateways\PayPal\Signup   $signup                [Global] The Signup class.
  * @var bool                                          $is_merchant_active    [Global] Whether the merchant is active or not.
  * @var bool                                          $is_merchant_connected [Global] Whether the merchant is connected or not.
+ * @var bool                                          $is_ssl                [Global] Whether the site is SSL or not.
```

Comment:
Although I might be mistaken, wouldn't it be more appropriate to pass in these variables instead of setting $is_ssl below?

Reply:
Also the `@since` needs to be updated.

Reply:
Would it make sense to check if the variable is empty that way is_ssl() isn't overriding the global variable?

Reply:
Regarding adding a test, you're right. I'll need to write this in a test-friendly way, too. Back to the drawing board. Thanks, @redscar!